app/code/Magento/InventoryCatalog/Model/UpdateLegacyStockItemByPlainQuery.php

```diff
@@ -3,48 +3,24 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
-declare(strict_types=1);
+declare(strict_types = 1);
```

Comment:
no need to use spaces
`types=1`
that currently breaks static tests

Reply:
done